install.py

```diff
@@ -433,6 +445,7 @@ def parse_commandline_args():
 	with warnings.catch_warnings():
 		warnings.simplefilter("ignore")
 		setup_log_stream(args)
+		install_python_package('distro')
```

Comment:
this breaks if pip isn't already installed on the system. Specifically for Python versions <= PY2.7.9 and <= PY3.4

pip is installed later on in `install_prerequisites` as of right now.

Reply:
@gavindsouza  How shall this be resolved then. 
we will not install the prerequisite if the system is not compatible. OR we install prerequisite before checking the OS.

Reply:
By installing pip first. I've pushed the changes to this branch.